base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -535,11 +630,7 @@ static int get_argument_type(WCHAR* argument)
     return OP_PARAM_INVALID;
 }
 
-/* FIXME
-argument T not supported
-*/
-
-static BOOL process_arguments(int argc, WCHAR* argv[])
+static BOOL process_arguments(int argc, WCHAR *argv[])
```

Comment:
Please revert. (except for the comment that has to go away now)

Reply:
I reverted.